brave/browser-laptop
# Add new "empty new tab" option in preferences (for what new tab does)

![screen shot 2016-11-16 at 12 13 47 pm](https://cloud.githubusercontent.com/assets/4733304/20361800/2e9824e4-abf6-11e6-8a11-de36d7392c2a.png)

*includes tests* :smile: 

Fixes https://github.com/brave/browser-laptop/issues/5682

Please note: choosing this option does NOT put focus in the URL bar. Is this OK?
cc: @diracdeltas

Auditors: @bradleyrichter, @bbondy

Test Plan:
1. Launch Brave and go to preferences
2. Change "A new tab shows" to "empty new tab"
3. Open a new tab; notice it's about:blank
